tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/OboTests2.cs

```diff
@@ -43,7 +43,7 @@ public class OboTests2
         private static readonly string[] s_publicCloudOBOServiceScope = { "api://23c64cd8-21e4-41dd-9756-ab9e2c23f58c/access_as_user" };
         private static readonly string[] s_arlingtonOBOServiceScope = { "https://arlmsidlab1.us/IDLABS_APP_Confidential_Client/user_impersonation" };
 
-        //TODO: acquire scenario specific client ids from the lab resonse
+        //TODO: acquire scenario specific client ids from the lab response
```

Comment:
Can we have an integration test (or modify some existing ones) around this new flow? I know it's well covered with unit tests, but it's helpful to have integration tests for high level API to be able to get Fiddler traces easily etc. 

I'm thinking: 

- one test where the AT is still valid + normal key
- one test where the AT is no longer valid (expire it or wipe it) + custom key

Reply:
Added these extra tests in https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2820/commits/f89db0b8abd37eeaf37088250881d09be26bdf8b